app/scripts/modules/core/pipeline/config/triggers/webhook/webhookTrigger.html

```diff
@@ -0,0 +1,32 @@
+<ng-form name="webhookForm">
+
+    <div class="row">
+      <div class="col-md-3 sm-label-right">
+        Webhook URL
+      </div>
+      <div class="col-md-6">
+        http://localhost:8089/webhooks/webhook</i>
+      </div>
+    </div>
+
+    <div class="form-group row">
+      <label class="col-md-3 sm-label-right">
+        <b>Constraints</b>
+        <help-field key="pipeline.config.webhook.constraints"></help-field>
+      </label>
+      <div class="col-sm-9">
+        <map-editor model="trigger.constraints" add-button-label="Add New Constraint" allow-empty="true"></map-editor>
```

Comment:
`<map-editor>` has an option to include an `on-change` property, which you could use to avoid having to manually include the `$scope.$watchCollection` in the controller:
```html
<map-editor model="trigger.constraints" add-button-label="Add New Constraint" allow-empty="true" on-change="vm.updateExamplePayload()"></map-editor>
```
(for that to work, you'd want to tweak the definition below:
```javascript
function updateExamplePayload () {
...
}
```
 to:
```javascript
this.updateExamplePayload = () => $scope.examplePayload = angular.copy($scope.trigger.constraints);
```